lib/module.js

```diff
@@ -64,7 +64,18 @@ function axiosModule (_moduleOptions) {
     debug: false,
     progress: true,
     proxyHeaders: true,
-    proxyHeadersIgnore: ['accept', 'host', 'x-forwarded-host', 'cf-ray', 'cf-connecting-ip', 'content-length', 'content-md5', 'content-type'],
+    proxyHeadersIgnore: [
+      'accept',
+      'cf-connecting-ip',
+      'cf-ray',
+      'content-length',
+      'content-md5',
+      'content-type',
+      'host',
+      'x-forwarded-host',
+      'x-forwarded-port',
+      'x-forwarded-prot'
```

Comment:
This seems to be duplicate

Reply:
It was `proto`, not `prot`.
I saw it immediately after pushing.
Sorry about that